src/components/DataCard/DataCard.tsx

```diff
@@ -0,0 +1,213 @@
+
+  const sourceUrl =
+    (post as any)?.recordUrl ??
+    (post as any)?.url ??
+    (post as any)?.source ??
+    sourceUrlFromId;
+
+  const handleDownload = async (url: string, name: string) => {
+    try {
+      const response = await fetch(url);
+      const blob = await response.blob();
+      const blobUrl = URL.createObjectURL(blob);
+      const link = document.createElement("a");
+      link.href = blobUrl;
+      link.download = name;
+      link.click();
+      URL.revokeObjectURL(blobUrl);
+    } catch (err) {
+      console.error("Erro ao baixar arquivo:", err);
+    }
+  };
+
+  return (
+    <div className="flex min-h-[176px] w-full flex-col gap-6 rounded-lg border border-[#EFEFEF] bg-[#F8F7F8] px-4 py-4 transition-transform duration-300 hover:scale-[1.01]">
+      <div className="flex flex-col gap-6 lg:flex-row lg:items-start lg:justify-between lg:gap-10">
+        <div className="flex flex-1 flex-col gap-4">
+          <div className="flex flex-col gap-3">
+            <h2 className="text-base font-semibold leading-4 text-[#292829]">
+              {post.title}
+            </h2>
+
+            {normalizedTags.length > 0 && (
+              <div className="flex flex-wrap gap-2">
+                {normalizedTags.map(({ key, label, color }) => (
+                  <span
+                    key={key}
+                    style={{
+                      backgroundColor: color ?? "#018F39",
+                      color: "#fff",
+                    }}
+                    className="min-h-[20px] min-w-[51px] rounded-full border border-[#E2E8F0] px-[10px] py-[2px] text-xs font-medium leading-4"
+                  >
+                    {label}
+                  </span>
+                ))}
+              </div>
+            )}
+          </div>
+
+          {post.description && (
+            <p className="text-base font-normal leading-[150%] text-[#292829]">
+              {post.description}
+            </p>
+          )}
+
+          <div className="flex flex-wrap items-center gap-3 text-xs font-normal leading-5 text-[#7E797B]">
+            <span>
+              Publicado em: {formattedDate} | Versão: {post.version || "N/A"}
+            </span>
+          </div>
+        </div>
+
+        {(sourceUrl || primaryFile) && (
+          <div className="flex w-full flex-row flex-wrap items-center justify-between gap-3 lg:w-auto lg:flex-nowrap lg:justify-end">
+            {sourceUrl && (
+              <a
+                href={sourceUrl}
+                target="_blank"
+                rel="noopener noreferrer"
+                className="flex min-w-[138px] items-center justify-center gap-2 rounded-md border border-[#DCDBDC] bg-white px-4 py-2 text-sm font-semibold text-[#038F39] transition-all hover:bg-emerald-50 hover:text-emerald-900 hover:scale-105"
+              >
+                <Icon id="icon-external" size={16} className="text-[#038F39]" />
+                Ir para fonte
+              </a>
```

Comment:
We usually use Link from next/link instead of <a tag, something like this: 
```suggestion
              <Link
                href={sourceUrl}
                target="_blank"
                className="flex min-w-[138px] items-center justify-center gap-2 rounded-md border border-[#DCDBDC] bg-white px-4 py-2 text-sm font-semibold text-[#038F39] transition-all hover:bg-emerald-50 hover:text-emerald-900"
              >
                <Icon id="icon-external" size={16} className="text-[#038F39]" />
                Ir para fonte
              </Link>
```